roles/populate_mirror_registry/defaults/main.yml

```diff
@@ -17,9 +17,10 @@ downloads_path: "{{ ansible_env.HOME }}"
 image_name_registry: ocpdiscon-registry
 
 # oc and opm install
-release_url: "https://mirror.openshift.com/pub/openshift-v{{ openshift_version_parts[0] }}/clients/ocp"
-oc_tar: openshift-client-linux.tar.gz
-opm_tar: opm-linux.tar.gz
+stable_release_url: "https://mirror.openshift.com/pub/openshift-v{{ openshift_version_parts[0] }}/clients/ocp/stable"
```

Comment:
Could you add a check into validate inventory like we did [here](https://github.com/redhat-partner-solutions/crucible/blob/main/roles/validate_inventory/tasks/vms.yml#L9-L17), that explains that `release_url` can no longer be used and the functionality around it has changed. As anyone overriding this would not get what they were expecting.

I don't think this neatly fits into any of the existing check files feel free to make a new one.

Reply:
Done, I hope it's fine now too.